auto-update/Control/AutoUpdate.hs

```diff
@@ -90,6 +92,11 @@ data UpdateSettings a = UpdateSettings
     -- Default: does nothing.
     --
     -- @since 0.1.0
+    , updateActionModify   :: Maybe (a -> IO a)
+    -- ^ Optional action to be performed to get the current value
+    -- and updating it if necessary.
```

Comment:
This sentence switches tense—I think you want "to get the current value and update it if necessary."